src/AzureMcp.csproj

```diff
@@ -101,4 +101,8 @@
       <ContentWithTargetPath Remove="@(FilesToRemove)" />
     </ItemGroup>
   </Target>
+
+  <PropertyGroup>
+    <TrimmerSingleWarn>false</TrimmerSingleWarn>
```

Comment:
this property needs to b enabled to produce detailed AOT compact warn, there is no `dotnet` command line option for this.